app/controllers/HomeController.java

```diff
@@ -1,21 +1,58 @@
 package controllers;
 
+import LunchManCore.Apprentice;
+import LunchManCore.FridayLunch;
+import LunchManCore.Rota;
 import play.mvc.*;
 
+import services.CSVHelper;
 import views.html.*;
 
-import java.util.Arrays;
+import java.time.LocalDate;
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
+
+import static services.CSVHelper.createScheduleFromCSV;
 
 /**
  * This controller contains an action to handle HTTP requests
  * to the application's home page.
  */
 public class HomeController extends Controller {
 
+    private String apprenticeCSV = "/Users/molliestephenson/Java/LunchMan/csvs/apprentices.csv";
+    private String scheduleCSV = "/Users/molliestephenson/Java/LunchMan/csvs/schedule.csv";
+
+    private Rota rota = new Rota(4, LocalDate.now());
+
+    public HomeController() {}
```

Comment:
Is this constructor needed? Seems that if you were to use it you would get `NullPointerException` immediately because the `apprenticeCSV` and `scheduleCSV` are not defined